src/components/SubscribeForm.js

```diff
@@ -1,28 +1,69 @@
 import React from 'react';
+import * as Sentry from '@sentry/browser';
 
 import { FormControl, FormLabel, Input, Stack, Text } from '@chakra-ui/core';
 import Button from './Button';
+import { toGlobalId } from '../utils/globalId';
+
+// Some configurable things
+const FORM_USER = 'a5fccdceb916e00a46d603ed8';
+const FORM_ID = '992a06eec8';
+export const FORM_ACTION = `https://rebuildblackbusiness.us10.list-manage.com/subscribe/post?u=${FORM_USER}&amp;id=${FORM_ID}`;
```

Comment:
Mad a few of these keys a bit more reusable

Reply:
Love the documentation for reasons of change!